assets/scss/news.scss

```diff
@@ -61,10 +61,10 @@
 
   .news-story-list-item-title {
     a {
-      color: $colour_metal_grey;
+      color: govuk-colour("dark-grey");
```

Comment:
Are these the same colour? And if not is the branding settings setting it back to metal grey. Just making sure theres no visible change.

Reply:
It is overridden, but this grey should probably be removed as:
- it is overridden by another style anyway
- it is a link, so grey is the wrong colour

So I'll kill it off.